modules/context/context.go

```diff
@@ -103,9 +103,11 @@ func (ctx *Context) Handle(status int, title string, err error) {
 
 	switch status {
 	case 404:
-		ctx.Data["Title"] = "Page Not Found"
+		ctx.Data["Title"] = title
 	case 500:
 		ctx.Data["Title"] = "Internal Server Error"
+	default:
+		ctx.Data["Title"] = title
```

Comment:
Why are the changes in this file needed?

Reply:
i think the status maybe not the num in (404, 500), and then it should be own default value

Reply:
I looked through the gitea source - there's no reference anywhere where a code different from 404 or 505 is used. Also - I'm referring to the previous change as well, why did `"Page Not Found"` suffice?